modules/rhizo_base/manifests/apt.pp

```diff
@@ -65,31 +80,35 @@
       key         => {
         'id'      => '68576280',
         'source'  => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
-       } 
+       }
```

Comment:
If the repo were not such a total mess already, I would ask to put WS changes or cosmetic clean ups in their own commit. Maybe going forward... :)